allennlp/data/fields/field.py

```diff
@@ -54,15 +54,15 @@ def count_vocab_items(self, counter: Dict[str, Dict[str, int]]):
         """
         pass
 
-    def human_readable_dict(self) -> JsonDict:
+    def human_readable_repr(self) -> object:
```

Comment:
Let's just make the return type `Any`. I thinks that's the canonical way to type it.

```suggestion
    def human_readable_repr(self) -> Any:
```